cadence/nfts/blind-edition-nft/transactions/create_editions.template.cdc

```diff
@@ -1,7 +1,14 @@
 import {{ contractName }} from {{{ contractAddress }}}
 
+/// This transaction creates a batch of new editions.
+///
+/// Parameters:
+/// - mintIDs: a list containing the mint ID for each edition.
+/// - limits: a list containing an optional size limit for each edition.
+///
 transaction(
-    sizes: [UInt64],
+    mintIDs: [String],
```

Comment:
Good callout. I thought about making it optional, since it's really just a utility to prevent duplicate mints. I'll follow up with that in a separate PR, though, since the non-blind edition contract uses this too.